test/unit-test/ota_utest.c

```diff
@@ -356,6 +381,8 @@ static void otaInit( const char * pClientID,
     pOtaAppBuffer.certFilePathSize = OTA_CERT_FILE_PATH_SIZE;
     pOtaAppBuffer.pStreamName = pOtaAppBuffer.pCertFilePath + pOtaAppBuffer.certFilePathSize;
     pOtaAppBuffer.streamNameSize = OTA_STREAM_NAME_SIZE;
+    pOtaAppBuffer.pUrl = pOtaAppBuffer.pStreamName + pOtaAppBuffer.streamNameSize;
```

Comment:
Is this corrrect? It seems odd that we are assigning a string to a stream name plus the size of the stream.

Reply:
I'm just making pointers to point to different offset in the buffer I declared above.